src/main/java/com/es/phoneshop/model/cart/cartService/CartService.java

```diff
@@ -1,14 +1,19 @@
 package com.es.phoneshop.model.cart.cartService;
```

Comment:
Mini-cart is not being displayed in case of error on the cart - the error is thrown, and MiniCartServlet is not being called.
That's ok for now.